lib/Solver/QueryLoggingSolver.h

```diff
@@ -0,0 +1,72 @@
+//===-- QueryLoggingSolver.h
+//---------------------------------------------------===//
```

Comment:
Strange line break

Reply:
Hah agreed! I'm just pulling things directly from upstream, so for future updates/diffing may as well keep it as consistent as possible.